src/utils.c

```diff
@@ -2182,3 +2182,18 @@ void utils_start_new_geany_instance(const gchar *doc_path)
 	else
 		g_printerr("Unable to find 'geany'");
 }
+
+
+void utils_slist_free_full(GSList *list, GDestroyNotify deleter)
+{
+#if GLIB_CHECK_VERSION(2, 28, 0)
```

Comment:
On 2015-06-14 05:20 AM, Colomban Wendling wrote:
> In src/utils.c
> <https://github.com/geany/geany/pull/513#discussion_r32379019>:
>
>> @@ -2182,3 +2182,18 @@ void utils_start_new_geany_instance(const gchar *doc_path)
>>  	else
>>  		g_printerr("Unable to find 'geany'");
>>  }
>> +
>> +
>> +void utils_slist_free_full(GSList *list, GDestroyNotify deleter)
>> +{
>> +#if GLIB_CHECK_VERSION(2, 28, 0)
>
> we depend on 2.28 now bdc082c
> <https://github.com/geany/geany/commit/bdc082cb7782828bd3be5992ca2e6006f2330890>

Ah right, nice. I actually looked only at this line:

https://github.com/geany/geany/commit/bdc082cb7782828bd3be5992ca2e6006f2330890#diff-67e997bcfdac55191033d57a16d1408aR77